[2025-04-10T22:16:41.625+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-10T22:16:41.647+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_ad_sets_run manual__2025-04-10T22:13:09.195693+00:00 [queued]>
[2025-04-10T22:16:41.655+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_ad_sets_run manual__2025-04-10T22:13:09.195693+00:00 [queued]>
[2025-04-10T22:16:41.656+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-04-10T22:16:41.672+0000] {taskinstance.py:2890} INFO - Executing <Task(DbtRunLocalOperator): dbt_processing.data_transformations.meta_ads_ad_sets_run> on 2025-04-10 22:13:09.195693+00:00
[2025-04-10T22:16:41.687+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1715) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-10T22:16:41.689+0000] {standard_task_runner.py:72} INFO - Started process 1726 to run task
[2025-04-10T22:16:41.689+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'meta_ads_elt_pipeline', 'dbt_processing.data_transformations.meta_ads_ad_sets_run', 'manual__2025-04-10T22:13:09.195693+00:00', '--job-id', '418', '--raw', '--subdir', 'DAGS_FOLDER/meta_ads_elt_pipeline.py', '--cfg-path', '/tmp/tmpv_dsxhul']
[2025-04-10T22:16:41.690+0000] {standard_task_runner.py:105} INFO - Job 418: Subtask dbt_processing.data_transformations.meta_ads_ad_sets_run
[2025-04-10T22:16:41.735+0000] {task_command.py:467} INFO - Running <TaskInstance: meta_ads_elt_pipeline.dbt_processing.data_transformations.meta_ads_ad_sets_run manual__2025-04-10T22:13:09.195693+00:00 [running]> on host ffa99f2ae5fe
[2025-04-10T22:16:41.810+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='meta_ads_elt_pipeline' AIRFLOW_CTX_TASK_ID='dbt_processing.data_transformations.meta_ads_ad_sets_run' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T22:13:09.195693+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-10T22:13:09.195693+00:00'
[2025-04-10T22:16:41.812+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-10T22:16:41.813+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-10T22:16:41.813+0000] {logging_mixin.py:190} INFO - Current task name:dbt_processing.data_transformations.meta_ads_ad_sets_run state:running start_date:2025-04-10 22:16:41.647378+00:00
[2025-04-10T22:16:41.813+0000] {logging_mixin.py:190} INFO - Dag name:meta_ads_elt_pipeline and current dag run status:running
[2025-04-10T22:16:41.813+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-10T22:16:42.846+0000] {local.py:213} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-04-10T22:16:42.847+0000] {local.py:416} INFO - Cloning project to writable temp directory /tmp/tmp8iudgau3 from /opt/airflow/dags/dbt/dbt_biera
[2025-04-10T22:16:42.848+0000] {local.py:427} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/meta_ads_elt_pipeline__dbt_processing__data_transformations/target/partial_parse.msgpack
[2025-04-10T22:16:42.855+0000] {config.py:360} INFO - Profile caching is enable.
[2025-04-10T22:16:42.862+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-10T22:16:42.864+0000] {config.py:340} INFO - Profile found in cache using profile: /tmp/cosmos/profile/c99bf486c96f06e0233408d45300d7cde96c22a7e21e1f49350115e2a86b810b/profiles.yml.
[2025-04-10T22:16:42.864+0000] {runner.py:60} INFO - Trying to run dbtRunner with:
 ['run', '--models', 'meta_ads_ad_sets', '--full-refresh', '--project-dir', '/tmp/tmp8iudgau3', '--profiles-dir', '/tmp/cosmos/profile/c99bf486c96f06e0233408d45300d7cde96c22a7e21e1f49350115e2a86b810b', '--profile', 'default', '--target', 'dev']
 in /tmp/tmp8iudgau3
[2025-04-10T22:16:42.932+0000] {logging_mixin.py:190} INFO - 22:16:42  Running with dbt=1.9.3
[2025-04-10T22:16:43.167+0000] {logging_mixin.py:190} INFO - 22:16:43  Registered adapter: postgres=1.9.0
[2025-04-10T22:16:43.855+0000] {logging_mixin.py:190} INFO - 22:16:43  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.dbt_biera.analytics
[2025-04-10T22:16:44.256+0000] {logging_mixin.py:190} INFO - 22:16:44  Found 4 models, 5 sources, 434 macros
[2025-04-10T22:16:44.258+0000] {logging_mixin.py:190} INFO - 22:16:44
[2025-04-10T22:16:44.259+0000] {logging_mixin.py:190} INFO - 22:16:44  Concurrency: 1 threads (target='dev')
[2025-04-10T22:16:44.260+0000] {logging_mixin.py:190} INFO - 22:16:44
[2025-04-10T22:16:44.375+0000] {logging_mixin.py:190} INFO - 22:16:44  1 of 1 START sql view model staging.meta_ads_ad_sets ........................... [RUN]
[2025-04-10T22:16:44.501+0000] {logging_mixin.py:190} INFO - 22:16:44  1 of 1 OK created sql view model staging.meta_ads_ad_sets ...................... [CREATE VIEW in 0.12s]
[2025-04-10T22:16:44.516+0000] {logging_mixin.py:190} INFO - 22:16:44
[2025-04-10T22:16:44.517+0000] {logging_mixin.py:190} INFO - 22:16:44  Finished running 1 view model in 0 hours 0 minutes and 0.25 seconds (0.25s).
[2025-04-10T22:16:44.565+0000] {logging_mixin.py:190} INFO - 22:16:44
[2025-04-10T22:16:44.566+0000] {logging_mixin.py:190} INFO - 22:16:44  Completed successfully
[2025-04-10T22:16:44.566+0000] {logging_mixin.py:190} INFO - 22:16:44
[2025-04-10T22:16:44.567+0000] {logging_mixin.py:190} INFO - 22:16:44  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-04-10T22:16:45.196+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.source.meta_ads_ad_sets is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-10T22:16:45.197+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/cosmos/operators/local.py:616: UserWarning: The dataset URI postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets is not AIP-60 compliant: URI format postgres:// must contain database, schema, and table names. In Airflow 3, this will raise an exception.
  datasets = [Dataset(uri) for uri in uris]

[2025-04-10T22:16:45.198+0000] {local.py:465} INFO - Inlets: [Dataset(uri='postgres://postgres:5432/warehouse.source.meta_ads_ad_sets', extra=None)]
[2025-04-10T22:16:45.198+0000] {local.py:466} INFO - Outlets: [Dataset(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets', extra=None)]
[2025-04-10T22:16:45.198+0000] {local.py:656} INFO - Assigning inlets/outlets with DatasetAlias
[2025-04-10T22:16:45.240+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-10T22:16:45.240+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=meta_ads_elt_pipeline, task_id=dbt_processing.data_transformations.meta_ads_ad_sets_run, run_id=manual__2025-04-10T22:13:09.195693+00:00, execution_date=20250410T221309, start_date=20250410T221641, end_date=20250410T221645
[2025-04-10T22:16:45.247+0000] {taskinstance.py:3064} INFO - Creating event for DatasetModel(uri='postgres://postgres:5432/warehouse.staging.meta_ads_ad_sets', extra={}) through aliases "meta_ads_elt_pipeline__data_transformations__meta_ads_ad_sets_run"
[2025-04-10T22:16:45.298+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-10T22:16:45.298+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-10T22:16:45.298+0000] {logging_mixin.py:190} INFO - Dag name:meta_ads_elt_pipeline queued_at:2025-04-10 22:13:09.216082+00:00
[2025-04-10T22:16:45.298+0000] {logging_mixin.py:190} INFO - Task hostname:ffa99f2ae5fe operator:DbtRunLocalOperator
[2025-04-10T22:16:45.350+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-10T22:16:45.359+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
